test/InquirerUnitTest/InputListUnitTest.cs

```diff
@@ -62,8 +62,41 @@ public void ShouldCallConsoleWriteLine()
             var input = new ListInput(name, message, new string[] { "option1", "option2" }, consoleRender);
             input.Render();
 
-            consoleRender.Received().Render(Arg.Any<string[]>(), Arg.Any<string[]>());
+            consoleRender.Received().RenderList(Arg.Any<ListInputMessage[]>(), Arg.Any<string[]>());
         }
 
+        [Fact]
+        public void FirstQuestionShoulBeADifferentColor()
+        {
+            var message = "Which option?";
+            var name = "option";
+            var options = new string[] { "option1", "option2" };
+            var consoleRender = Substitute.For<IScreenManager>();
+            var input = new ListInput(name, message, options, consoleRender);
+
+            var questions = input.GetQuestion();
+            var inputMessages = input.ToListInputMessages(questions);
+
+            inputMessages[1].Message.Should().Be("> option1");
+            inputMessages[1].IsSelected.Should().Be(true);
+            inputMessages[1].ConsoleColor.Should().Be(ConsoleColor.Cyan);
+        }
+
+        [Fact]
+        public void FirstQuestionNotShoulBeADifferentColor()
```

Comment:
I think here is related to the Second question and not First, right?

Reply:
Yes, you are correct! The method name refers to the second item in the list.